doc/examples/timeouts.rst

```diff
@@ -0,0 +1,80 @@
+
+.. _timeout-example:
+
+Client Side Operation Timeout
+=============================
+
+PyMongo 4.2 introduced :meth:`~pymongo.timeout` and the ``timeoutMS``
+URI and keyword argument to :class:`~pymongo.mongo_client.MongoClient`.
```

Comment:
Could this paragraph mention the motor version that bumped our req to pymongo>=4.2?

Reply:
Done